plugins/mapper-annotated-text/src/yamlRestTest/resources/rest-api-spec/test/mapper_annotatedtext/20_synthetic_source.yml

```diff
@@ -0,0 +1,164 @@
+---
```

Comment:
Can we also test a few failure scenarios? Especially because using synthetic source depends on multiple settings combined together (stored, multi-field). 

Also I am wondering what happens if we have a field that is both stored and has the keyword multi-filed. I guess in that case we would prefer relying on the stored field because it would allow us to return the contents exactly as they are...including duplicates.

Reply:
Failure scenarios are tested in `TextFieldFamilySyntheticSourceSupport#invalidExample`. 

If field is stored, we always prefer that over a multi field. I'll add a test.